gocourse11/filter/filter.go

```diff
@@ -0,0 +1,35 @@
+package filter
+
+import (
+	"gocourse11/aquarium"
+)
+
+//import "gocourse11/aqurium"
+// Потрібно вирахувати рівень забруднення і запустити системи фільтрації в зміненому режимі,
+//також додати до води необхідні хімічні речовини, які покращать її стан.
+
+type System struct{}
+
+func (System) CalculatePollution(a aquarium.Aquarium) int {
+	var n, mg int
+	for _, v := range a.Chemichals {
+		switch v {
+		case aquarium.N:
+			n += 1
+		case aquarium.Mg:
+			mg += 1
```

Comment:
```suggestion
			mg++
```

https://go.dev/ref/spec#IncDec_statements